frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -124,15 +136,52 @@ type DispatchProps = ExtractReturn<typeof mapDispatchToProps>;
 
 type Props = {| ...DispatchProps |};
 
-type State = { hoveredListItem: ?number };
+type State = { hoveredListItem: ?number, isComputingMeshfile: boolean };
 
 class MeshesView extends React.Component<Props, State> {
+  intervalID: ?TimeoutID;
+
   state = {
     hoveredListItem: null,
+    isComputingMeshfile: false,
   };
 
   componentDidMount() {
     maybeFetchMeshFiles(this.props.segmentationLayer, this.props.dataset, false);
+    this.fetchJobData();
+  }
+
+  async fetchJobData(): Promise<void> {
+    const jobs = await getJobs();
+    const wasComputingMeshfile = this.state.isComputingMeshfile;
+    const isComputingMeshfile =
+      jobs.filter(
+        job =>
+          job.type === "compute_mesh_file" &&
+          job.datasetName === this.props.datasetName &&
+          job.state === "STARTED",
+      ).length > 0;
+    if (wasComputingMeshfile && !isComputingMeshfile) {
+      Toast.info(
+        <React.Fragment>
+          A meshfile generation job for this dataset has finished. Click{" "}
+          <a href="/jobs" target="_blank">
+            here
+          </a>{" "}
+          to see all jobs.
```

Comment:
```suggestion
          The requested computation of a mesh file for this dataset has finished. You can now use the "Load Precomputed Mesh" in the "Meshes" tab.
```

Reply:
I'm not sure this sends the right message. We currently don't check, whether it's the requested computation. It only indicates, that there was some running meshfile job for this dataset, and now there's not. That's why I included the link to the jobs, so one could see how the job ended.
We would need to save the job id when starting a job and then check with the job id, if the job is still running, was successful or failed. That's possible, but not the case right now.

Reply:
Good point! How about:

```suggestion
          The computation of a mesh file for this dataset has finished. You can now use the "Load Precomputed Mesh" in the "Meshes" tab.
```

This doesn't imply that the user actually requested that mesh file and simply announces that the feature is available now. What do you think? In my opinion, there is no great use in delegating the user to the jobs page, since the exciting stuff happens in the mesh tab :)